src/CommunityToolkit.Maui.MediaElement/EventArgs/MediaFailedEventArgs.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace CommunityToolkit.Maui.MediaElement;
+
+public class MediaFailedEventArgs : EventArgs
+{
+	/// <summary>
+	/// Gets a description of why the media failed to load and/or play.
+	/// </summary>
+	public string ErrorMessage { get; } = string.Empty;
+
+	internal MediaFailedEventArgs(string errorMessage)
```

Comment:
Do we want this to be `internal`? IMO this can be bad in terms of extensibility, if devs need to add more functions to their media element control.

Reply:
(I haven't reviewed the PR yet, just chiming in after seeing this notification)

If we do decide to not make the constructor public, I recommend changing `internal` to [`protected private`](https://learn.microsoft.com/dotnet/csharp/language-reference/keywords/private-protected) like we did for [`BaseCommand` in XCT](https://github.com/xamarin/XamarinCommunityToolkit/blob/b55322ac95e863347cc25a9b029b3a812e98682e/src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Internals/BaseCommand.shared.cs#L25-L29).

#### Background story

In XCT, I created `BaseCommand` to share logic between `AsyncCommand` and `AsyncValueCommand`, and I had to make `BaseCommand` a `public class` because `AsyncCommand` and `AsyncValueCommand` are both `public`. 

But, I didn't want us supporting `BaseCommand` as a public API because that is outside the scope of the `AsyncCommand` + `AsyncValueCommand` APIs. So I used `protected private` + [`[EditorBrowsable(EditorBrowsableState.Never)]`](https://github.com/xamarin/XamarinCommunityToolkit/blob/b55322ac95e863347cc25a9b029b3a812e98682e/src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Internals/BaseCommand.shared.cs#L12) to allow us to inherit from it internally, but it prevents devs from consuming it externally.

Reply:
Yeah that all makes sense. I tend to work as most closed by default. Anything that is public is something we need to maintain and I'd rather be convinced after the fact on why someone wants this than to make it public and later on have to make a breaking change.

But this one makes sense and doesn't seem too bad. Thanks!